packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -510,12 +510,11 @@ export async function getPageStaticInfo(params: {
     }
 
     let extraConfig: Record<string, any> | undefined
+    extraConfig = {}
```

Comment:
Seems this can be simplified:
```suggestion
    let extraConfig: Record<string, any> = {}
```

Reply:
Good catch